_projects/dendrite.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "Contribute to Dendrite"
+desc: "Dendrite is a work in progress matrix homeserver written in Go, help implement some aspect of the Matrix Federation API"
+difficulty: "Ranges from Easy through to Hard depending on the API!"
+issues:
+ - https://github.com/matrix-org/dendrite/issues/671
+mentors:
+ - erikjohnston
+ - richvdh
+initiatives:
+ - GSoC
+tags:
+ - Server
+ - Federation
+---
+
+[Dendrite](https://github.com/matrix-org/dendrite) is a work in progress matrix homeserver written in Go. Being a large project there are many areas of the homeserver APIs that a GSoC project could work on, there is a [spreadsheet](https://docs.google.com/spreadsheets/d/1tkMNpIpPjvuDJWjPFbw_xzNzOHBA-Hp50Rkpcr43xTw) that tracks the current implementation status of the various APIs. A good GSOC proposal would use this and discussion with mentors and the community to find an appropriate and realistic subset of these APIs to work on during the project.
```

Comment:
There are better resources for this now right?

Reply:
Yes, https://github.com/matrix-org/dendrite/projects/2, good call